web/styles/settings.css

```diff
@@ -1987,3 +1987,8 @@ $option_title_width: 180px;
         margin-top: -10px;
     }
 }
+
+#edit_user_full_name.invalid {
+    border: 1px solid hsl(3deg 73% 74%);
+    box-shadow: 0 0 2px hsl(3deg 73% 74%);
+}
```

Comment:
How does this look in the dark theme? Usually we use a color variable for this sort of detail.

Reply:
I used the values already used for an invalid message in the text area.
You can see how it looks in the dark theme on the video in the PR.